.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
-        language_version: python3.7
+        language_version: python3
   - repo: https://github.com/asottile/seed-isort-config
```

Comment:
@SamWilliams-dev - do you know why the python version constraint was relaxed?

Reply:
Ok, @SamWilliams-dev  replied to indicate he needed to relax the constraint since he's running 3.9.x on his dev machine.

3.9 is the currently supported version of Python - so there is merit in adopting it. Since we deploy python in the `Pepys` distributable we could upgrade with minimal impact to users.